.circleci/config.yml

```
@@ -194,6 +211,7 @@ workflows:
     jobs:
       - docs
       - lint
+      - coverage
```

I think we should only run the `coverage` job if the tests pass:

```suggestion
      - coverage:
          requires:
            - test
```

---

Is there any downside of running it anyway?

---

I think it can generate weird reports, imagine that something breaks during the tests (a panic) and only part of the tests has been run, probably the coverage is going to be not-realistic at all. I think I agree with @mgdelacroix here, I prefer only run coverage if tests passes.